pytripgui/app_logic/app_callbacks.py

```diff
@@ -191,7 +191,12 @@ def open_voxelplan_callback(self, patient_item):
 
         patient = patient_item.data
         patient.open_ctx(filename + ".ctx")  # Todo catch exceptions
```

Comment:
Reading CTX cube should also work without extensions:
https://github.com/pytrip/pytrip/blob/master/pytrip/cube.py#L398

Reply:
Reading VDX two lines later requires an extension, so we should leave it as-is for consistency.